src/Quartz.Tests.Integration/Quartz.Tests.Integration.csproj

```diff
@@ -70,6 +70,26 @@
     </Reference>
     <Reference Include="System" />
     <Reference Include="System.Core" />
+    <Reference Include="FakeItEasy, Version=1.25.2.0, Culture=neutral, PublicKeyToken=eff28e2146d5fd2c, processorArchitecture=MSIL">
```

Comment:
Ah, good catch. We don't need these. I think VS made these changes automatically because I was changing the oracle dependency through VS's interface (instead of directly in the project.json, as would have been better). I've reverted these changes (in the Quartz.Tests.Unit.csproj file, as well).